### PR TITLE
268 tooltip

### PR DESCRIPTION
**Pull request by DavydiukDmytro**

### CODE REVIEW

File: client/src/components/ellipsisText/style.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { cn } from '@/utils';
+
+import { IGetStylesProps } from './types';
+
+export const getStyles = ({ isEllipsisTooltip, classNameWrapper }: IGetStylesProps) => ({
+  toolTipWrapper: cn(
+    'absolute z-[99999] overflow-hidden w-fit rounded border-[1px] border-skyBlue px-2 py-1 bg-white',
+    classNameWrapper,
```
Comment (Wolshebnik) [inline client/src/components/ellipsisText/style.ts:8]: rename to className

File: client/src/components/ellipsisText/index.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+import { getStyles } from './style';
+import { IEllipsisTextProps } from './types';
+
+export const EllipsisText = ({
+  content,
+  children,
+  margin = 0,
+  delay = 200,
+  isShowAlways,
+  classNameWrapper,
```
Comment (Wolshebnik) [inline client/src/components/ellipsisText/index.tsx:27]: rename to className

File: client/src/components/ellipsisText/types.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export interface IEllipsisTextProps {
+  delay?: number;
+  margin?: number;
+  isShowAlways?: boolean;
+  children: React.ReactNode;
+  classNameWrapper?: string;
```
Comment (Wolshebnik) [inline client/src/components/ellipsisText/types.ts:6]: rename to className

File: client/src/components/employee-card/info/index.tsx [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 import { useTranslations } from 'next-intl';
 
+import { EllipsisText } from '@/components/ellipsisText';
```
Comment (Wolshebnik) [inline client/src/components/employee-card/info/index.tsx:5]: import { EllipsisText } from '@/components'

**Review by Wolshebnik — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**